files/ja/orphaned/web/javascript/reference/global_objects/asyncfunction/prototype/index.md

```diff
@@ -27,55 +27,7 @@ original_slug: Web/JavaScript/Reference/Global_Objects/AsyncFunction/prototype
 
 ## ブラウザー実装状況
 
-{{CompatibilityTable}}
```

Comment:
This also removed the deprecated `CompatibilityTable` macro.

Reply:
Although, as this is in an orphaned directory we could just delete the file?

Reply:
This file can be deleted, and it should be redirected to "Web/JavaScript/Reference/Global_Objects/AsyncFunction".

Reply:
> This file can be deleted, and it should be redirected to "Web/JavaScript/Reference/Global_Objects/AsyncFunction".

Should we do it as part of this PR or open an issue to do it post landing this one? Also, last I heard the `delete` function of Yari is broken?